Frontend/CO.CDP.OrganisationApp/Pages/CookieDetails.cshtml

```diff
@@ -8,116 +8,162 @@
+            <td class="govuk-table__cell">When you close your browser</td>
+        </tr>
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">.AspNetCore.Session</th>
+            <td class="govuk-table__cell">@StaticTextResource.Supplementary_CookieDetails_TableRow_Cookies</td>
+            <td class="govuk-table__cell">When you close your browser</td>
+        </tr>
+    </tbody>
+</table>
 
-    <p class="govuk-body">@StaticTextResource.Supplementary_CookieDetails_BannerInfo</p>
+<h3 class="govuk-heading-s">@StaticTextResource.Supplementary_CookieDetails_BannerMessages</h3>
 
-    <table class="govuk-table">
-        <thead class="govuk-table__head">
-            <tr class="govuk-table__row">
-                <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Name</th>
-                <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Purpose</th>
-                <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Expires</th>
-            </tr>
-        </thead>
-        <tbody class="govuk-table__body">
-            <tr class="govuk-table__row">
-                <th scope="row" class="govuk-table__header">.AspNetCore.Culture</th>
-                <td class="govuk-table__cell">@StaticTextResource.Supplementary_CookieDetails_TableRow_Culture</td>
-                <td class="govuk-table__cell">1 year</td>
-            </tr>
-            <tr class="govuk-table__row">
-                <th scope="row" class="govuk-table__header">SIRSI_COOKIES_PREFERENCES_SET</th>
-                <td class="govuk-table__cell">@StaticTextResource.Supplementary_CookieDetails_TableRow_Preferences</td>
-                <td class="govuk-table__cell">1 year</td>
-            </tr>
-        </tbody>
-    </table>
+<p class="govuk-body">@StaticTextResource.Supplementary_CookieDetails_BannerInfo</p>
 
-    <h2 class="govuk-heading-m">@StaticTextResource.Supplementary_CookieDetails_ChangeSettings</h2>
-    <p class="govuk-body">@Html.Raw(StaticTextResource.Supplementary_CookieDetails_ChangeSettingsInfo)</p>
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Name</th>
+            <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Purpose</th>
+            <th scope="col" class="govuk-table__header">@StaticTextResource.Supplementary_CookieDetails_TableHeader_Expires</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">FT_COOKIES_ PREFERENCES_SET</th>
```

Comment:
```suggestion
            <th scope="row" class="govuk-table__header">FT_COOKIES_PREFERENCES_SET</th>
```

Reply:
I see this is how they've got it written on FTS, to force a line break, I'll resolve this.